src/main/java/org/sonarlint/intellij/issue/IssuePointer.java

```diff
@@ -28,24 +31,34 @@
 import javax.annotation.Nullable;
 import java.util.concurrent.atomic.AtomicLong;
 
-public class IssuePointer {
+public class IssuePointer implements Trackable {
   private static final AtomicLong UID_GEN = new AtomicLong();
   private final long uid;
   private final RangeMarker range;
   private final Issue issue;
   private final PsiFile psiFile;
-  private final long creationDate;
+  private long creationDate;
+  private final Integer checksum;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "checksum" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1068)